exercises/practice/largest-series-product/src/test/groovy/LargestSeriesProductSpec.groovy

```diff
@@ -0,0 +1,212 @@
+import spock.lang.*
+
+class ExerciseSpec extends Specification {
+    @IgnoreRest
+    def "finds the largest product if span equals length"() {
+        expect:
+        // Please check the "largestProduct" property
```

Comment:
We probably should remove the please check, expect, and use comments. They’re apparently intended for the test suite author, not the student. The comments don’t provide information that isn’t in the test case itself so they’re noisy.

Reply:
Updated test generator in such a way that it can generate all the code, not just stubs with comments. Comments are not generated anymore.